### ISSUE TITLE
KFF crash on MC 1.20.4 with MinecraftForge 49.0.8

### ISSUE THREAD

**Issue by Prototik** [thedarkcolour/KotlinForForge#89]
```
[13Dec2023 14:19:03.511] [modloading-worker-0/ERROR] [thedarkcolour.kotlinforforge.LoggerKt/LOADING]: Failed to register Automatic Kotlin subscribers. ModID: kotlinforforge, class thedarkcolour.kotlinforforge.test.KotlinForForge
java.lang.NoClassDefFoundError: net/minecraftforge/fml/loading/moddiscovery/ModAnnotation$EnumHolder
	at thedarkcolour.kotlinforforge.AutoKotlinEventBusSubscriber.<clinit>(AutoKotlinEventBusSubscriber.kt:40) ~[kfflang-4.8.0.jar%239!/:4.8.0]
	at thedarkcolour.kotlinforforge.KotlinModContainer.constructMod(KotlinModContainer.kt:74) ~[kfflang-4.8.0.jar%239!/:4.8.0]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:123) ~[fmlcore-1.20.4-49.0.3.jar!/:1.0]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraftforge.fml.loading.moddiscovery.ModAnnotation$EnumHolder
	at net.minecraftforge.securemodules.SecureModuleClassLoader.loadClass(SecureModuleClassLoader.java:447) ~[securemodules-2.2.7.jar!/:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at net.minecraftforge.securemodules.SecureModuleClassLoader.loadClass(SecureModuleClassLoader.java:422) ~[securemodules-2.2.7.jar!/:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	... 10 more
```

That comes from [huge commit](https://github.com/MinecraftForge/MinecraftForge/commit/6f9f0f0cfacc74d62be39359d773d5dda220ad73#diff-f5ef591e861c8b8910f09a4b87d6046b29d123f0af03b781b283711fd7eeba49) from Lex

**Comment by thedarkcolour** [comment]
Which mod did you use to cause this crash? I want to know what mod to test my fix with.

**Comment by Prototik** [comment]
None, KFF alone is enough to crash on latest forge due to removed classes

**Comment by thedarkcolour** [comment]
Fixed in 4.9.0
